tests/script-based-pre/stubbing_compiler_sessions/expected

```diff
@@ -0,0 +1 @@
+VERIFICATION:- SUCCESSFUL
```

Comment:
It'd be nice to add a small English description of the goal for this test. I don't know if that is the standard for Kani, but [here](https://github.com/diffblue/cbmc/blob/develop/regression/contracts-dfcc/assigns_enforce_01/test.desc) is an example from the CBMC repository.

Reply:
The `expected` file should be a subset of the Kani output. It doesn't accept comments or special characters. The only special handling we do is `\` at the end of a line.

Reply:
Good to know! So I guess we keep the descriptions in the test itself. Got it!